app/src/main/java/comcmput301f16t01/github/carrier/SearchActivity.java

```diff
@@ -42,6 +43,9 @@ public void onClick(DialogInterface dialogInterface, int i) {
                 Toast.makeText(activity, query, Toast.LENGTH_SHORT).show();
                 // TODO do the search with the inputted keyword query
                 // TODO consider any input handling on the keyword?
+                // Link to MockSearchResultsActivty
+                Intent intent = new Intent(SearchActivity.this, MockSearchResultsActivity.class);
```

Comment:
So Bennett has actually already implemented a real search results activity, and you can post real requests on elastic search and search for real requests too. Once you fix your merge conflicts, you'll see it. Please replace the mock stuff with some real stuff.